examples/demo13.py

```diff
@@ -0,0 +1,378 @@
+    logger.info('Read in %d galaxies from catalog'%len(cat))
+    # Just use a few galaxies, to save time.  Note that we are going to put 4000 galaxy images into
+    # our big image, so if we have n_use=10, each galaxy will appear 400 times.  Users who want a
+    # more interesting image with greater variation in the galaxy population can change `n_use` to
+    # something larger that evenly divides 4000 (but it should be <=500, which is the number of
+    # galaxies in the catalog).  The code is not set up to handle values of n_use that do not evenly
+    # divide n_tot, though it would not be very hard to do so.  With 4000 galaxies in a 4k x 4k
+    # image with the WFIRST pixel scale, the effective galaxy number density is 74/arcmin^2
+    n_use = 10
+    n_tot = 4000
+    n_per_gal = n_tot / n_use
+
+    # Here we carry out the initial steps that are necessary to get a fully chromatic PSF.  We use
+    # the getPSF() routine in the WFIRST module, which knows all about the telescope parameters
+    # (diameter, bandpasses, obscuration, etc.).  Note that we are going to arbitrarily choose a
+    # single SCA (Sensor Chip Assembly) rather than all of them, for faster calculations, and we're
+    # going to use a simple representation of the struts for faster calculations.  To do a more
+    # exact calculation of the chromaticity and pupil plane configuration, remove the
+    # `approximate_struts` and the `n_waves` keyword from the call to getPSF():
+    use_SCA = 7 # This could be any number from 1...18
+    logger.info('Doing expensive pre-computation of PSF.')
+    t1 = time.time()
+    PSFs = wfirst.getPSF(SCAs=use_SCA, approximate_struts=True, n_waves=10)
+    PSF = PSFs[use_SCA]
+    t2 = time.time()
+    logger.info('Done PSF precomputation in %.1f seconds!'%(t2-t1))
+
+    # Define the size of the postage stamp that we will use for each individual galaxy within the
+    # larger image, and for the PSF images.
+    stamp_size = 128
+
+    # We are going to choose a particular (RA, dec) location on the sky for our observation.
+    ra_targ = 30.*galsim.degrees
+    dec_targ = -10.*galsim.degrees
+    targ_pos = galsim.CelestialCoord(ra=ra_targ, dec=dec_targ)
+    ang = 0.*galsim.degrees
+    # Get the WCS for an observation at this position, with the focal plane array oriented at an
+    # angle of `ang` with respect to North.  The output of this routine is a list of WCS objects,
+    # one for each SCA.  We will then take the WCS for the SCA that we are going to use.
+    wcs_list = wfirst.getWCS(ang, pos=targ_pos, PA_is_FPA=True)
+    wcs = wcs_list[use_SCA]
+    # We need to find the center position for this SCA.  We'll tell it to give us a CelestialCoord
+    # corresponding to (X, Y) = (wfirst.n_pix/2, wfirst.n_pix/2).
+    SCA_cent_pos = wcs.toWorld(galsim.PositionD(wfirst.n_pix/2, wfirst.n_pix/2))
+
+    # We are going to just randomly distribute points in (X, Y).
```

Comment:
Again, present tense would read better here.  There are a few other times in the script where you use this pattern, which I won't keep harping on.  Just search for going and you should find them.